CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
 
 ## Next
 
+### Added dependency
+* postgres citext dependency added to enable aliases to be globally unique
```

Comment:
Should this PR also update the system-requirements website doc to include this dependency?